Plugin/BlankCanvas/HEADER.txt

```diff
@@ -0,0 +1,31 @@
+This file is part of ${name}, licensed under the BSD 3-Clause License
```

Comment:
It seems like the common license for Sponge projects(including elsewhere in cookbook) and many plugins is MIT, might be worthwhile to use that as the example license instead.